scripts/build_avado.sh

```diff
@@ -80,6 +80,11 @@ if [[ "${avado_version}" = "0.100.0" && "${release_id}" = "master-goerli" ]]; th
   upstream_version="master-goerli"
 fi
 
+# For debug-deploy builds, we need to use special upstream version, since we do not publish 0.100.0 Docker tag
```

Comment:
This `if` fragment is equivalent to the previous one, we could extend if condition to accept master-goerli or debug-goerli and in the content of the if, assign the release_id `upstream_version="${release_id}"`

Reply:
Adapted accordingly.